### PR TITLE
initial graybeard dialogue

### PR DESCRIPTION
**Pull request by SteveHNH**
First pass! 

Having issues witht he active quest logic, but I can figure it out

### CODE REVIEW

**Review by josejulio — APPROVED** [review]
great dialog

File: Dialogue/lordgraybeard.tres [hunk 1]
```diff
@@ -0,0 +1,1418 @@
...
+	=> END
+
+~ turn_in
+Lord Graybeard: All done then?
+Hero: You bet!
+Lord Graybeard: Well good! I guess the system is in better shape than it was.
+Lord Graybeard: I can feel it in my bits.
+Hero: Gross.
+Lord Graybeard: Anyway, here are your hours. Now go home!
+# need to have a function call to award hours and end the game"
```
Comment (josejulio) [inline Dialogue/lordgraybeard.tres:134]: `increase_hours(100)` should do. It's part of the game_state

Comment (SteveHNH) [reply Dialogue/lordgraybeard.tres:134]: Great! I'll add that. man, that's convenient

**Review by aleccohan — APPROVED** [review]
Looking great!  The only issue I got was for some reason in the title_screen.tscn I had to change the "dialogue" thing

**Pull request merged**